source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -39,6 +65,25 @@ class ACRDateField: NSView, InputHandlingViewProtocol {
         }
     }
     
+    var value: String {
+        guard let selectedDate = selectedDate else {
+            return ""
+        }
```

Comment:
Here you should check whether textfield is empty.
```suggestion
        guard !textfield.stringValue.isEmpty, let selectedDate = selectedDate else {
            return ""
        }
```